scripts/build_ci.bash

```diff
@@ -10,6 +10,8 @@ sudo apt-get install check
 sudo apt-get install bats
 sudo pip install coverxygen
 sudo ln -s /bin/systemctl /usr/bin/systemctl
+sudo systemctl mask swupd-update.service
+sudo systemctl mask swupd-update.timer
```

Comment:
If you mask the timer and service you aren't testing if the disabled is really working.
I checked the current error and systemctl is falling with message:

```
External command: Created symlink /home/runner/work/swupd-client/swupd-client/aup-basic_AUT001/testfs/target-dir/etc/systemd/system/swupd-update.service → /dev/null.
External command: Created symlink /home/runner/work/swupd-client/swupd-client/aup-basic_AUT001/testfs/target-dir/etc/systemd/system/swupd-update.timer → /dev/null.
External command: Failed to stop swupd-update.service: Unit swupd-update.service not loaded.
External command: Failed to stop swupd-update.timer: Unit swupd-update.timer not loaded.
```

Did you try to initialize the service instead of masking it?

Reply:
@otaviobp : You were right...placing starting the service unit files inside and starting the swupd-update.timer helped it. I did need the mask.